src/main/java/io/confluent/connect/jdbc/source/JdbcSourceConnectorConfig.java

```diff
@@ -89,6 +92,16 @@
   public static final boolean NUMERIC_PRECISION_MAPPING_DEFAULT = false;
   private static final String NUMERIC_PRECISION_MAPPING_DISPLAY = "Map Numeric Values By Precision";
 
+  public static final String NUMERIC_MAPPING_CONFIG = "numeric.mapping";
+  private static final String NUMERIC_MAPPING_DOC =
+          "Whether or not to attempt mapping NUMERIC values by precision to integral types";
```

Comment:
So it seems like this needs more detail, and it's no longer "Whether or not to attempt...".